app/webpacker/components/Tickets/TicketWorkbenches/EditPersonTicketWorkbench.jsx

```diff
@@ -18,7 +19,7 @@ function EditPersonValidations({ ticketDetails }) {
   if (error) return <Errored />;
 
   return validators.dob.map((validator) => (
-    <Message warning>{validator.message}</Message>
+    <Message warning>{I18n.t(`validators.person.${validator.id}`)}</Message>
```

Comment:
This will not fill the i18n keys correctly, I'm afraid

Reply:
I.e. the message will look like "The date of birth of [missing 'name' param] is on January 1st"

Reply:
Yes, actually the same error is there in production as well. Since this is internal to WRT, I thought of adjusting with this error because knowing 'something is wrong' is better than knowing nothing.

TBH I didn't spend time to check if name can be populated here. I guess it should be possible through ticketDetails. I'll explore if that is possible.

Reply:
I just checked and WCA ID is available. For now I have added WCA ID. But I feel this fix should be separate PR, I'll do that later. For now I've added it in this PR, will change later.